common/src/main/java/gov/cms/ab2d/common/model/PdpClient.java

```diff
@@ -11,12 +11,16 @@
 import javax.validation.constraints.NotNull;
 import java.util.*;
 
+/**
+ * Represents a PDP organization which possesses a single set of credentials for
+ * accessing the system.
+ */
 @Entity
 @Table(name = "user_account")
 @Getter
 @Setter
 @EqualsAndHashCode(onlyExplicitlyIncluded = true, callSuper = false)
-public class User extends TimestampBase implements UserDetails {
+public class PdpClient extends TimestampBase implements UserDetails {
```

Comment:
I love getting rid of one of the **worst** domain nouns ever, replacing it with something that actually matches the domain.